Spanner/tests/System/PgWriteTest.php

```diff
@@ -297,6 +300,8 @@ public function arrayFieldComplexValueProvider()
             [$this->randId(), 'arraytimestampfield', [new Timestamp(new \DateTime),null,new Timestamp(new \DateTime)]],
             [$this->randId(), 'arraydatefield', [new Date(new \DateTime),null,new Date(new \DateTime)]],
             [$this->randId(), 'arraypgnumericfield', [new PgNumeric("0.12345"),null,new PgNumeric("12345")]],
+            [$this->randId(), 'arraypgjsonbfield', [new PgJsonb('{"a":1.1,"b":"hello"}'),null,
+            new PgJsonb(["a" => 1, "b" => null])]],
```

Comment:
nit: would it be better to add  `new PgJsonb('{}'), new PgJsonb([])` as well?

Reply:
Done!